app/web/template/layout/attorney-summary.gohtml

```diff
@@ -0,0 +1,75 @@
+{{ define "attorney-summary" }}
+    {{range $i, $a := .Attorneys}}
+        {{ $attorneyNumber := inc $i }}
+
+        <h2 class="govuk-heading-m">Attorney {{ $attorneyNumber }}</h2>
+        <dl class="govuk-summary-list">
+            <div class="govuk-summary-list__row" id="name-{{ $attorneyNumber }}">
+                <dt class="govuk-summary-list__key">
+                    {{ tr $.App "name" }}
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    {{ $a.FirstNames }} {{ $a.LastName }}
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link" href="{{ link $.App $.DetailsPath }}?id={{ $a.ID }}&from={{ $.App.Page }}&#f-first-names">
+                        {{ trFormatHtml $.App "changeNameLinkText" "FirstNames" $a.FirstNames "LastName" $a.LastName }}
+                    </a>
+                </dd>
+            </div>
+            <div class="govuk-summary-list__row" id="date-of-birth-{{ $attorneyNumber }}">
+                <dt class="govuk-summary-list__key">
+                    {{ tr $.App "dateOfBirth" }}
+                </dt>
+                <dd class="govuk-summary-list__value">
+                    {{ formatDate $a.DateOfBirth }}
+                </dd>
+                <dd class="govuk-summary-list__actions">
+                    <a class="govuk-link" href="{{ link $.App $.DetailsPath }}?id={{ $a.ID }}&from={{ $.App.Page }}&#f-date-of-birth">
+                        {{ trFormatHtml $.App "changeDOBLinkText" "FirstNames" $a.FirstNames "LastName" $a.LastName }}
+                    </a>
+                </dd>
+            </div>
+            <div class="govuk-summary-list__row govuk-!-padding-top-1" id="email-{{ $attorneyNumber }}">
```

Comment:
I don't see the difference the top padding is adding here?

Reply:
Ah good spot - this was left in from when I was trying some different layouts. Will remove.